src/Testcontainers.WebDriver/WebDriverBuilder.cs

```diff
@@ -0,0 +1,219 @@
+namespace Testcontainers.WebDriver;
+
+/// <inheritdoc cref="ContainerBuilder{TBuilderEntity, TContainerEntity, TConfigurationEntity}" />
+[PublicAPI]
+public sealed class WebDriverBuilder : ContainerBuilder<WebDriverBuilder, WebDriverContainer, WebDriverConfiguration>
```

Comment:
@kiview @eddumelendez I am interested in your opinion on the builder configuration and the test. Since TC for .NET does not have the `dependsOn` feature, I think the implementation looks very good. I am particularly uncertain about the APIs that rely on environment variables. It is the old discussion about which APIs we expose. WDTY?

Reply:
Setting env vars in the container configuration couple the implementation with the image itself. Unless, of course, there is certain logic to do with those as in other implementations. I think those can be removed `WithConfigurationOptions`, `WithJavaEnvironmentOptions`, `SettingScreenResolution`, `SetSessionTimeout`, `SetTimeZone` because can be easily added with `WithEnvironment(k, v)`. It is the same case as in Azurite implementation IIRC.

Reply:
In tc-java, currently, there is no usage of `dependsOn` between `BrowserWebDriverContainer` and `VncContainer`. `VncContainer` is started manually once the `BrowserWebDriverContainer` started.

Reply:
> Setting env vars in the container configuration couple the implementation with the image itself. Unless, of course, there is certain logic to do with those as in other implementations.

Indeed. I have the same opinion.

> In tc-java, currently, there is no usage of `dependsOn` between `BrowserWebDriverContainer` and `VncContainer`. `VncContainer` is started manually once the `BrowserWebDriverContainer` started.

I see. In any case, once we have the `dependsOn` feature, implementation will be even simpler.